lib/services/channel.js

```diff
@@ -0,0 +1,43 @@
+var Service = require('./service');
+
+/**
+ * Service for interacting with the channel endpoints on the Beam REST API.
+ * @access public
+ * @augments {Service}
+ */
+function ChannelService () {
+    Service.apply(this, arguments);
+}
+
+ChannelService.prototype = new Service();
+
+/**
+ * Retrieves a list of all channels.
+ * @param  {Object} data
+ * @param  {Object} data.page The page of results to get.
+ * @param  {Object} data.limit Number of results per page to retrieve.
+ * @return {Promise}
```

Comment:
:+1: I've started doing this as well. This package is mostly documented via jsdoc, so it's important here. You can use `@typedef` to define the attributes of the channels that will be returned.